zulip_bots/zulip_bots/bots/define/define.py

```diff
@@ -66,7 +68,8 @@ def get_bot_define_response(self, original_content):
 
             except Exception as e:
                 response += self.REQUEST_ERROR_MESSAGE
-                logging.exception(e)
+                # logging.exception expects a string
+                logging.exception(str(e))
```

Comment:
I know what's causing this, in Python3, you have to pass in a str to `logging.exception`. I would recommend removing the comment you added and doing `logging.exception("")` instead. `logging.exception` should take care of embedding the exception info in the logged message automatically! :)